src/platform/plugins/shared/charts/public/services/palettes/palettes.test.tsx

```diff
@@ -223,7 +237,7 @@ describe('palettes', () => {
         expect(color1).toEqual(color2);
       });
 
-      it('should return the same index of the behind text palette for same key', () => {
+      itIf(legacy)('should return the same index of the behind text palette for same key', () => {
```

Comment:
nit: why not use a simple condition here?
```js
if(legacy){
  it('...')
```

Reply:
Idk, I wanted something like [playwright](https://playwright.dev/docs/test-annotations#conditionally-skip-a-test) allows...

```ts
test('skip this test', async ({ page, browserName }) => {
  test.skip(browserName === 'firefox', 'Still working on it');
});
```

Reply:
Removed in 893053224dbdfd64c14a732a7c0688357fa7e7d3